shopping_app/lib/providers/super_hero_provider.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:convert';
+import 'package:flutter/material.dart';
+import 'package:http/http.dart' as http;
+import '../models/superhero.dart';
+
+class SuperHeroProvider with ChangeNotifier {
+  List<Superhero> itemHero;
+
+  List<Superhero> get(heros) {
+    return [...heros];
```

Comment:
Why we are using this?

Reply:
```suggestion
    return [...heros];
```
This is returning A complete list of heroes in the List<Model> list.